snapcraft/internal/project_loader/_extensions/kde_neon.py

```diff
@@ -48,14 +62,24 @@ def __init__(self, *, extension_name: str, yaml_data: Dict[str, Any]) -> None:
         platform_snap = _PLATFORM_SNAP[yaml_data.get("base")]
         self.root_snippet = {
             "plugs": {
+                "icon-themes": {
+                    "interface": "content",
+                    "target": "$SNAP/data-dir/icons",
+                    "default-provider": "gtk-common-themes:icon-themes",
```

Comment:
Please take a look at #2763

Reply:
I removed the slot specification but I don't see any difference in the resulting snap. It seems snapcraft strips the slot specification before writing `snap.yaml`.

Is there a bug report or more information about @kenvandine his PR?